### PR TITLE
Pub 468: Refactor combined_sequences

### PR DESCRIPTION
**Pull request by subutai**
Cleanup scripts for combined sequences experiments

### CODE REVIEW

**Comment by subutai** [comment]
@lscheinkman If you look at combined_sequences.py it gives you an idea of what I'm doing here. I should be able to get rid of sequence_convergence.py and object_convergence.py.

I'm not done yet but any feedback welcomed.

File: projects/combined_sequences/combined_sequences.py [hunk 1]
```diff
@@ -223,203 +236,97 @@ def runExperiment(args):
...
   args.update({"statistics": exp.statistics})
 
-  # Can't pickle experiment so can't return it for batch multiprocessing runs.
-  # However this is very useful for debugging when running in a single thread.
-  # if plotInferenceStats:
-  #   args.update({"experiment": exp})
   return args
 
 
 def runExperimentPool(numSequences,
```
Comment (lscheinkman) [inline projects/combined_sequences/combined_sequences.py]: This function is not being called anymore. Do you still need it?

Comment (subutai) [reply projects/combined_sequences/combined_sequences.py]: Yes, I will soon add experiments that need it.

**Comment by subutai** [comment]
@lscheinkman Ready for review - no rush.

File: projects/combined_sequences/generate_plots.py [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+        fields=[
+          ("L4 PredictedActive", "Predicted active cells in sensorimotor layer"),
+          ("TM NextPredicted", "Predicted cells in temporal sequence layer"),
+          ("TM PredictedActive",
+           "Predicted active cells in temporal sequence layer"),
+        ],
+        basename="pure_sequences",
+        trialNumber=trialNum,
+        plotDir=os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                             "detailed_plots")
```
Comment (lscheinkman) [inline projects/combined_sequences/generate_plots.py]: Why the 2 different plots folders (`plots` vs `detailed_plots`)?

Comment (subutai) [reply projects/combined_sequences/generate_plots.py]: The `detailed_plots` folder contains tons of individual plots.

**Pull request merged**